s3/pkg/service/object.go

```diff
@@ -355,45 +358,77 @@ func (s *s3Service) GetObject(ctx context.Context, req *pb.GetObjectInput, strea
 		return err
 	}
 
-	var finalReader io.Reader
-	finalReader = reader
+	allBytes, _ := ioutil.ReadAll(reader)
+
+	log.Infof("allBytes.len - %v", len(allBytes))
+
+	allDecBytes := make([]byte, 0)
+
 	if bucket.ServerSideEncryption.SseType == "SSE" {
-		finalReader, _ = utils.WrapAlignedEncryptionReader(reader, 0,
-			bucket.ServerSideEncryption.EncryptionKey,
-			bucket.ServerSideEncryption.InitilizationVector)
-	}
+		// this should match the upload chunk size during multipart upload, which is 5MB (5*1024*1024). During multipart
```

Comment:
5MB is not specific, it's depends on client, dashboard use 5MB as default.

Reply:
yes, that is why used 5 MB here. One thing i could think of was, to update the API for ```initMultipart``` to take in the chunk size as an input, persist it and use it here

Reply:
Is it necessary to change the encryption chunk according to the part size? How about using a fixed chunk size independent of it?

Reply:
thought of that, but ruled it out, since, it will get messy during a multi part upload...example, if we encrypt each 2 MB chunk, and multipart chunks each 5MB, it is a problem